spec/fixtures/simple.rb

```diff
@@ -1,3 +1,4 @@
+sleep(0.1)
```

Comment:
In the future this should be changed to:

* connect
* **add breakpoint**
* play
* assert that it's been paused on breakpoint
* and that **step in/out/over** is enabled

I don't think no one wants flaky tests :)